src/mbgl/util/utf.hpp

```diff
@@ -2,20 +2,18 @@
 
 #include <memory>
 
-#include <locale>
 #include <codecvt>
+#include <locale>
```

Comment:
Removed. So `clang-format` will put includes in alphabetical order... Is our style "add includes in alphabetical order, but don't reorder existing ones"?